src/Assets/ugui-mvvm/Converters/BoolInverter.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Globalization;
+using UnityEngine;
+
+namespace uguimvvm.converters
+{
+    /// <summary>
+    /// Toggles the passed in boolean value 
```

Comment:
```suggestion
    /// Inverts the passed in boolean value 
```

"Toggles" sounds like something that responds to user input.